lib/services/android-project-service.ts

```diff
@@ -260,8 +256,16 @@ export class AndroidProjectService extends projectServiceBaseLib.PlatformProject
 	public async interpolateData(projectData: IProjectData, platformSpecificData: IPlatformSpecificData): Promise<void> {
 		// Interpolate the apilevel and package
 		this.interpolateConfigurationFile(projectData, platformSpecificData);
+		const appResourcesDirectoryPath = projectData.getAppResourcesDirectoryPath();
+
+		let stringsFilePath: string;
+
+		if (this.$androidResourcesMigrationService.hasMigrated(appResourcesDirectoryPath)) {
+			stringsFilePath = path.join(this.getAppResourcesDestinationDirectoryPath(projectData), constants.MAIN_DIR, constants.RESOURCES_DIR, 'values', 'strings.xml');
```

Comment:
Can extract `this.getAppResourcesDestinationDirectoryPath(projectData)` to a variable outside the if statements.